Content.Shared/PAI/PAIComponent.cs

```diff
@@ -31,6 +31,13 @@ public sealed partial class PAIComponent : Component
     [DataField("midiAction", serverOnly: true)] // server only, as it uses a server-BUI event !type
     public EntityUid? MidiAction;
 
+    [DataField("mapActionId", serverOnly: true,
+        customTypeSerializer: typeof(PrototypeIdSerializer<EntityPrototype>))]
+    public string? MapActionId = "ActionPAIOpenMap";
+
+    [DataField("mapAction", serverOnly: true)] // server only, as it uses a server-BUI event !type
+    public EntityUid? MapAction;
+
```

Comment:
The entity is still being added on the client.

Reply:
I'm taking a stab in the dark here, but do I need to remove the `serverOnly: true`? I stole that from the midi playing action, because in all honesty I don't really understand the code yet.